text/0013-schema-enums.md

```diff
@@ -0,0 +1,97 @@
+# Enums in Schemas
+
+## Related issues and PRs
+
+- Reference Issues: 
+- Implementation PR(s): 
+
+## Timeline
+
+- Start Date: 2023-06-30
+- Date Entered FCP:
+- Date Accepted:
+- Date Landed:
+
+## Summary
+
+Add support for enumerations of `bool`, `string`, and `long` types in schemas, and use this information when doing policy validation and schema-based-parsing.
```

Comment:
I imagine enumerations will be most useful for strings, so I think we should consider restricting the scope of this RFC to only `string`s.
* Enumerations could possibly be useful for `long`s too (e.g. for error codes), but that case can also be encoded using `string`s. The `long` type doesn't necessarily give an advantage because I doubt you would want to do `long`-specific operations on the values in the enum (e.g., addition, less-than comparison). 
* I don't think enums will be useful for `bool` (what is the point of a field that always has to be `true`?).

If I'm wrong, and you had specific use cases in mind, then please add them to the RFC.

Reply:
Mike had a use case for bools, I'll add that. Longs just seems intuitive, but you're right you could model it with strinsg as well

Reply:
I'm curious about the use case for bools.  I don't see the harm in supporting longs and bools in addition to strings.

Reply:
Added use cases